mmv1/products/container/api.yaml

```diff
@@ -354,6 +354,18 @@ objects:
         description: |
           The name of the Google Compute Engine network to which the cluster is
           connected. If left unspecified, the default network will be used.
+      - !ruby/object:Api::Type::NestedObject
```

Comment:
For handwritten resources, you don't have to update the api.yaml file if there is one associated.

This file isn't used for Terraform, it was used in other tools like Ansible/Inspec and we've since dropped support of those tools. Might as well delete this block

Reply:
I reverted the version.